### PR TITLE
adding cancel reset test

### PR DESCRIPTION
**Pull request by lalon4**

### CODE REVIEW

File: discovery-infra/tests/test_cancel_and_reset.py [hunk 1]
```diff
@@ -90,3 +90,48 @@ def test_cancel_reset_after_node_boot(self, api_client, node_controller, cluster
...
+        # Cancel cluster install once cluster installation start
+        self.wait_for_installing_in_progress(cluster_id=cluster_id, api_client=api_client)
+        self.cancel_cluster_install(cluster_id=cluster_id, api_client=api_client)
+        assert self.is_cluster_in_cancelled_status(
+            cluster_id=cluster_id,
+            api_client=api_client
+        )
+        # Shutdown one node
+        nodes = node_controller.list_nodes()
+        node = list(nodes.values())[0]
```
Comment (lalon4) [inline discovery-infra/tests/test_cancel_and_reset.py:116]: this one depending on https://github.com/openshift/assisted-test-infra/pull/214/files
will fix it once it merged

File: discovery-infra/tests/test_cancel_and_reset.py [hunk 2]
```diff
@@ -90,3 +90,48 @@ def test_cancel_reset_after_node_boot(self, api_client, node_controller, cluster
...
+        node.shutdown()
+        # Reset cluster install
+        self.reset_cluster_install(cluster_id=cluster_id, api_client=api_client)
+        assert self.is_cluster_in_insufficient_status(
```
Comment (nshidlin) [inline discovery-infra/tests/test_cancel_and_reset.py:120]: Isn't the cluster in insufficient status regardless of whether or not a node is shutdown? Is there a way to see why it's in that status and assert on that?

**Pull request merged**